keyboards/cans12er/cans12er.h

```diff
@@ -0,0 +1,16 @@
+#ifndef CANS12ER_H
```

Comment:
QMK uses the `#pragma once` include guard for header files, instead of `#ifndef`/`#define`/`#endif`.

```suggestion
#pragma once
```